### PR TITLE
Upgrade API

### PR DESCRIPTION
**Pull request by JacobMGEvans**
copilot:all

### CODE REVIEW

**Review by nobrayner — DISMISSED** [review]
Looks about right. Lets see what blows up I guess :joy:

**Comment by JacobMGEvans** [comment]
> Looks about right. Lets see what blows up I guess 😂

AHAHAHAHA That was only the start

File: schema.prisma [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+model Contributors {
+    user            String     @unique
+    userId          Int        @id @unique
+    commits         Int
+    avatarUrl       String
+    deletions       Int
+    additions       Int
+    raidStatsId     String?
+    RaidStats       RaidStats? @relation(fields: [raidStatsRaidId], references: [raidId])
+    raidStatsRaidId String?
```
Comment (nobrayner) [inline schema.prisma]: I don't really get what is going on here... Is a Contributor tied to a single raidstat?

Comment (JacobMGEvans) [reply schema.prisma]: Yes, RaidStat is a Raid, Contributors are all the people on a Raid

**Review by nobrayner — COMMENTED** [review]
Looking great 🙌

**Review by nobrayner — COMMENTED** [review]
:+1:

**Comment by JacobMGEvans** [comment]
copilot:all

**Pull request closed without merging**